visualization/motion_planning_tasks/src/task_panel.cpp

```diff
@@ -148,23 +152,22 @@ void TaskPanel::addSubPanel(SubPanel* w, const QString& title, const QIcon& icon
  * will never be called if the display is disabled...
  */
 
-void TaskPanel::request(rviz::WindowManagerInterface* window_manager) {
-	++DISPLAY_COUNT;
-
-	rviz::VisualizationFrame* vis_frame = dynamic_cast<rviz::VisualizationFrame*>(window_manager);
-	if (SINGLETON || !vis_frame)
-		return;  // already defined, nothing to do
-
-	QDockWidget* dock =
-	    vis_frame->addPanelByName("Motion Planning Tasks", "moveit_task_constructor/Motion Planning Tasks",
-	                              Qt::LeftDockWidgetArea, true /* floating */);
-	assert(dock->widget() == SINGLETON);
+void TaskPanel::request(rviz_common::WindowManagerInterface* window_manager) {
+	//	++DISPLAY_COUNT;
```

Comment:
What about this part?

Reply:
It's because RViz2 doesn't expose VisualizationFrame class for the user and I don't know how to port it, in the demos I manually added the task panel and display